examples/serialization/load.py

```diff
@@ -0,0 +1,27 @@
+import chainer
+import numpy as np
+import model
+
+
+n_in = 784
+n_units = 100
+n_out = 10
```

Comment:
How about including these parameters in `model.py`?
We should demonstrate minimal code repetition is required in save/load scripts.